src/mdl/isoscelles/isoquant_output_to_scanpy_seurat_input.py

```diff
@@ -0,0 +1,123 @@
+import csv
+from collections import defaultdict
+from pathlib import Path
+from typing import Sequence
+
+import h5py
+import numpy as np
+import scipy.io
+import sparse
+
+import pandas as pd
+import shutil
+import pysam
+import os
+import gzip
+
+import scanpy as sc
+
+def isoquant_matrix(
+    isoquant_path: str | Path,
+    bam_path: str | Path,
+    valid_assignments=("unique", "unique_minor_difference"),
+    barcode_list: Sequence[int] = None,
+    feature_list: Sequence[tuple[str, str]] = None,
+    out_isoform_seurat_path: str = "isoforms_seurat",
+    out_h5ad_path: str = "isoforms_h5ad",
+    gene_Ensembl_to_name: str = "gene_Ensembl_to_name.tsv"
```

Comment:
this path is not going to work when the package is installed somewhere. It should just be a required argument